hexa/plugins/connector_dhis2/views.py

```diff
@@ -142,6 +200,68 @@ def data_element_detail(
     )
 
 
+def organisation_unit_detail(
```

Comment:
I think it would be nice to have some of the query logic in queryset methods (nice to have)

Reply:
Good point